core/src/components/input/input.tsx

```diff
@@ -432,37 +434,17 @@ export class Input implements ComponentInterface {
     this.isComposing = false;
   };
 
-  private clearTextInput = (ev?: Event) => {
+  private onClearButtonClick = (ev?: Event) => {
```

Comment:
Renamed this function as it is the click handler for the button and should not be re-used directly by other parts of the implementation. 

Instead if inner behavior needs to be shared, there should be another private function that both implementations call.